suite-common/wallet-core/src/send/sendFormSolanaThunks.ts

```diff
@@ -236,8 +259,18 @@ export const composeSolanaTransactionFeeLevelsThunk = createThunk<
             }));
 
         const resultLevels: PrecomposedLevels = {};
+
+        assertIsSolanaAccount(account);
+
         const response = predefinedLevels.map(level =>
-            calculate(account.availableBalance, output, level, tokenInfo),
+            calculate(
+                account.availableBalance,
+                output,
+                level,
+                decimals,
+                account.misc.rent!,
```

Comment:
And are we sure it can't be undefined?

Reply:
Yes, for Solana accounts, the misc property is always defined and the rent is too. Sadly the assertion has to be somewhere because the typing is not as precise as it could be, this is fragile I agree. 

We might want not to assert but fail gracefully to some default rent value so if this breaks, Solana still works, and only the rent-related logic would be "turned off". This makes sense to me, what do you think?

Reply:
Yeah I think maybe just `?? 0` should be enough probably